packages/gestalt/src/__snapshots__/Controller.jsdom.test.js.snap

```diff
@@ -6,9 +6,9 @@ exports[`Controller renders 1`] = `
     className="container"
     style={
       Object {
-        "left": -10,
+        "left": 4,
```

Comment:
Are these snapshot changes expected?

Reply:
They were, but you made me rethink about them. I just made a change that will prevent them, and make the implementation even safer in Pinboard. They are related to the new constant `skipNoEdgeCondition` that was needed for ScrollableContainer (flyout could not work as expected if placed in the top/left edge of the container without margins), but I made it conditional to having a ScrollableContainer as well.

Reply:
FIXED